vnext/Mso/crash/verifyElseCrash.h

```diff
@@ -11,10 +11,14 @@
 DECLSPEC_NORETURN void CrashWithRecovery(uint32_t tag) noexcept;
 
 #define DisableVecAssert() false
+#ifndef VerifyElseCrash
```

Comment:
Do we accidentally include this twice, or was this just some polish? Wondering if we could make the change in more upstream Mso.

Reply:
It was an annoying warning because we also have these macros in the MS.RN.Cxx. I do not want the MS.RN.Cxx depend on Mso just because of these two macros. In the MS.RN.Cxx we already had #ifndef/#endif, but if Mso and MS.RN.Cxx are included in a different order, then we see the warnings. Changing Mso code fixes the issue.
Overall, it feels that we must be super careful with the macros because they are global and we never know when they popup. E.g. the GetCurrentTimer Windows macro is conflicting with a member function in one of the XAML animation interface. Windows is better to drop use of these macros in favor of inline functions and 'if constexpr'.